### PR TITLE
#149 fix cache

### PR DESCRIPTION
**Pull request by ahmed-shehata**
#149 fix cache

### CODE REVIEW

**Comment by ahmed-shehata** [comment]
👍

File: src/index.js [hunk 1]
```diff
@@ -38,6 +38,8 @@ const AtlasSDK = {
   configure(options = {}) {
     const env = options.is_sandbox ? 'staging' : 'production';
     const fileName = `${options.client_id}-${env}`;
+    const cachedConfName = `${options.client_id}-${options.sales_channel}-${env}`;
```
Comment (ricardomatias) [inline src/index.js]: Could you explain hows does this help fixing the issue?

**Comment by Etchs** [comment]
👍

File: src/index.js (hunk 1 above)
Comment (ahmed-shehata) [reply src/index.js]: Already explained: https://github.com/zalando-incubator/atlas-js-core/issues/149

**Comment by ahmed-shehata** [comment]
👍

**Comment by Etchs** [comment]
👍

**Comment by Gregsen** [comment]
:+1:

**Pull request merged**